packages/schematics/angular/migrations/update-10/update-dependencies.ts

```diff
@@ -34,15 +41,21 @@ export default function (): Rule {
         version,
         overwrite: true,
       });
+
+      hasChanges = true;
+    }
+
+    if (hasChanges) {
+      context.addTask(new NodePackageInstallTask());
     }
 
     // Check for @angular-devkit/schematics and @angular-devkit/core
     for (const name of ['@angular-devkit/schematics', '@angular-devkit/core']) {
       const current = getPackageJsonDependency(host, name);
       if (current) {
         context.logger.info(
-          `Package "${name}" found in the workspace package.json. ` +
-            'This package typically does not need to be installed manually. ' +
+          `Package "${name}" found in the workspace package.json.` +
+            'This package typically does not need to be installed manually.' +
```

Comment:
These strings should include a trailing space in order to have a space between sentences.

Also, there is a spelling error in the title: `depedencies`.